### PR TITLE
Added UCMC service charts for preview, aat and demo

### PR DESCRIPTION
**Pull request by pd-dudkiewicz**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by pd-dudkiewicz** [comment]
> is ucmc a different team to cmc?
> 
> otherwise it should be cmc-unspecified-service
> 
> i.e. `<product>-<component>`

@timja yes, unspecified civil money claim is separate from CMC

File: k8s/aat/common/ucmc/ucmc-service.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ucmc-service
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ucmc/service:latest
+      aadIdentityName: ucmc
```
Comment (timja) [inline k8s/aat/common/ucmc/ucmc-service.yaml:24]: this should be in the helm chart

Comment (pd-dudkiewicz) [reply k8s/aat/common/ucmc/ucmc-service.yaml]: done

File: CODEOWNERS [hunk 2]
```diff
@@ -266,3 +266,7 @@ k8s/**/cluster-*/pcq/pcq-consolidation-service.yaml @hmcts/pcq-admins
 ### RSE
 k8s/demo/common/rse-istio-poc/** @hmcts/rse
 k8s/aat/common/rse/check.yaml @hmcts/rse
+
+### UCMC
+k8s/**/common/ucmc/namespace.yaml @hmcts/ucmc-admins
+k8s/**/common/ucmc/ucmc-service.yaml @hmcts/ucmc-admins
```
Comment (timja) [inline CODEOWNERS:272]: ** is only allowed once you're allowed into prod
```suggestion
k8s/aat/common/ucmc/ucmc-service.yaml @hmcts/ucmc-admins
```

File: CODEOWNERS [hunk 3]
```diff
@@ -266,3 +266,9 @@ k8s/**/cluster-*/pcq/pcq-consolidation-service.yaml @hmcts/pcq-admins
 ### RSE
 k8s/demo/common/rse-istio-poc/** @hmcts/rse
 k8s/aat/common/rse/check.yaml @hmcts/rse
+
+### UCMC
+k8s/demo/common/ucmc/namespace.yaml @hmcts/ucmc-admins
```
Comment (timja) [inline CODEOWNERS:271]: you can leave namespace as ** but not a blocker

Comment (pd-dudkiewicz) [reply CODEOWNERS]: Good point, we'll probably be adding more files there soon. Changed

File: CODEOWNERS [hunk 4]
```diff
@@ -266,3 +266,7 @@ k8s/**/cluster-*/pcq/pcq-consolidation-service.yaml @hmcts/pcq-admins
 ### RSE
 k8s/demo/common/rse-istio-poc/** @hmcts/rse
 k8s/aat/common/rse/check.yaml @hmcts/rse
+
+### UCMC
+k8s/demo/common/ucmc/** @hmcts/ucmc-admins
```
Comment (timja) [inline CODEOWNERS:271]:
```suggestion
k8s/**/common/ucmc/namespace.yaml @hmcts/ucmc-admins
```

File: CODEOWNERS [hunk 5]
```diff
@@ -266,3 +266,7 @@ k8s/**/cluster-*/pcq/pcq-consolidation-service.yaml @hmcts/pcq-admins
...
+k8s/aat/common/ucmc/** @hmcts/ucmc-admins
```
Comment (timja) [inline CODEOWNERS:272]:
```suggestion
k8s/aat/common/ucmc/ucmc-service.yaml @hmcts/ucmc-admins
```

**Pull request merged**